sunpy/tests/figure_tests_env_py36.yml

```diff
@@ -0,0 +1,26 @@
+name: sunpy-figure-tests-3.6
```

Comment:
Is it possible to keep the requirements the same as before, but just bump the python version? Then we can start bumping e.g. Matplotlib requirements when I have image comparison working and we can see what the changes are.

Reply:
I didn't think to do that. I just updated my current env, saved it to a file and then updated the versions manually. 

The image comparison I used was the figures on your repo, everything bar the issue I opened looked the same dispite the hash change.

Reply:
I did as you suggested. The hashes are the same but new python version.